package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "space-y",
+  "version": "1.0.0",
+  "description": "Space Y game",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack serve --mode development --open --hot ",
+    "build": "webpack --mode production",
+    "lint": "eslint . --fix --ext .tsx && stylelint --fix **/*.scss"
```

В проекте будут не только реакт-компоненты, но и обычные `.ts` файлы, поэтому лучше сделать так:

```suggestion
    "lint": "eslint . --fix --ext .ts,.tsx && stylelint --fix **/*.scss"
```

https://eslint.org/docs/user-guide/command-line-interface